### PR TITLE
feat(safe-claiming-app): new safe app to claim SAFE tokens

### PR DESCRIPTION
**Pull request by schmanu**
## What does this PR add
A new Safe App: Safe Claiming App

This app guides users and guardians through  their Safe Token claiming process.
It consists of 3 Steps:
- Educational Series
- Delegation of voting power
- Claiming

## How to test it
`yarn test`

## Screenshots
gnosis-safe io_app_rin_0x4f99325f843524dbedc853dE85D6F184F60C250A_apps_appUrl=http___localhost_3000_safe-claiming-app (1)
gnosis-safe io_app_rin_0x4f99325f843524dbedc853dE85D6F184F60C250A_apps_appUrl=http___localhost_3000_safe-claiming-app

### CODE REVIEW

**Comment by schmanu** [comment]
I have read the CLA Document and I hereby sign the CLA

File: apps/safe-claiming-app/src/components/steps/Status/usePendingTx.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { TransactionStatus } from "@gnosis.pm/safe-react-gateway-sdk"
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk"
+
+const POLL_INTERVAL = 3000
```
Comment (mikhailxyz) [inline apps/safe-claiming-app/src/components/steps/Status/usePendingTx.ts:5]: The block time is 13-14 seconds, I wonder if we can increase the interval to remove some load from the backend

**Review by mikhailxyz — COMMENTED** [review]
sehr gut

File: apps/safe-claiming-app/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "short_name": "safe-claiming-app",
+  "name": "$SAFE Claiming App",
+  "description": "Claim your $SAFE Tokens",
+  "iconPath": "",
+  "icons": [
```
Comment (dasanra) [inline apps/safe-claiming-app/public/manifest.json:6]: We are missing an icon in the manifest

Is there anything expected?

File: apps/safe-claiming-app/src/components/steps/Status/usePendingTx.ts (hunk 1 above)
Comment (schmanu) [reply apps/safe-claiming-app/src/components/steps/Status/usePendingTx.ts:5]: I'll increase it to 10 seconds.

File: apps/safe-claiming-app/public/manifest.json (hunk 2 above)
Comment (schmanu) [reply apps/safe-claiming-app/public/manifest.json:6]: I'm discussing it. It will most likely be the safe token icon. I'll add it when decided :)

Comment (schmanu) [reply apps/safe-claiming-app/public/manifest.json:6]: The safe token logo was added!

**Review by dasanra — APPROVED** [review]

**Pull request merged**